File: Easy/3408-count-the-number-of-special-characters-i/count-the-number-of-special-characters-i.py
class Solution:
    def numberOfSpecialChars(self, w: str) -> int:
        lower=[0]*26
        upper=[0]*26

        for ch in w:
            if ch.islower():
                lower[ord(ch)-ord('a')] += 1
            else:
                upper[ord(ch)-ord('A')] += 1

        res=0
        for i in range(26):
            if lower[i]>0 and upper[i]>0:
                res+=1

        return res
        

File: Easy/3408-count-the-number-of-special-characters-i/count-the-number-of-special-characters-i.java
class Solution{
    public int numberOfSpecialChars(String w){
        int n=w.length();

        int[] lower=new int[26];
        int[] upper=new int[26];
        for(char ch: w.toCharArray()){
            if(Character.isLowerCase(ch)){
                lower[ch-'a'] +=1;
            }
            else{
                upper[ch-'A'] += 1;
            }
        }

        int res=0;
        for(int i=0; i<26; i++){
            if(lower[i]>0 && upper[i]>0){
                res+=1;
            }
        }

        return res;

    }
}